docs/social-messaging/index.md

```diff
@@ -1,35 +1,35 @@
 # Introduction to the RingCentral Social Messaging APIs
 
+{! mdx_includes/social-messaging-beta-notice.md !}
+
 ## Overview
 
-RingCentral Social Messaging is an integrated social networking platform that consolidates various popular social media sites such as WhatsApp, Facebook, LinkedIn, and others into a centralized social messaging hub. Its primary aim is to offer RingCentral business customers an outstanding social media marketing solution.
+RingCentral Social Messaging is an integrated social networking platform that consolidates various popular social/business messaging apps like WhatsApp, Apple messages for business, etc and social media websites such as Facebook, Instagram, Twitter and others into a centralized social messaging hub. Its primary aim is to offer RingCentral business customers an outstanding social media marketing solution.
 
-Any RingCentral customer with RingCX (formerly known as Engage Digital) and MVP accounts will gain access to the Social Messaging platform through APIs. This enables RingCentral customers to either develop stand-alone applications or seamlessly integrate the social messaging service into their existing products, thereby enhancing user experience and productivity.
+Any RingCentral [RingEX customer](https://www.ringcentral.com/office/plansandpricing.html) can access the Social Messaging platform through our open APIs. This enables RingCentral customers to either develop stand-alone applications or seamlessly integrate the social messaging service into their existing products, thereby enhancing user experience and productivity.
 
-The Social Messaging API set is a part of the RingCentral API platform. Thus, building a social messaging application that would invoke the APIs will require an application registration on the RingCentral developer portal. If you're unfamiliar with RingCentral application development, please consult [the developer guide](https://developers.ringcentral.com/guide/applications) for assistance and guidance. To access the social messaging features via APIs, the application must be assigned the 'social-messaging' app scope, along with any other app scopes required to access other the MVP platform features.
+The Social Messaging API set is a part of the RingCentral API platform. Thus, building a social messaging application that would invoke the APIs will require an application registration on the RingCentral developer portal. If you're unfamiliar with RingCentral application development, please consult [the developer guide](https://developers.ringcentral.com/guide/applications) for assistance and guidance. To access the social messaging features via APIs, the application must be assigned the 'social-messaging' app scope, along with any other app scopes required to access other the RingEX platform features.
```

Comment:
I believe "Social Messaging" app scope is internal so it may not appear for the developers in Dev Portal. Also the scope is "Social Messaging" not "social-messaging".